.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Expected behaviour
+
+_Explain what you expected to happen._
+
+### Actual behaviour
+
+_Explain what actually happened. If an exception occurred, please include a stack trace if available._
+
+### Steps to reproduce
+
+_A concise, repeatable, example of how to illustrate the issue._
```

Comment:
Being picky and possibly wrong but should this be _A concise and repeatable example of how to illustrate the issue_?

Reply:
This is mainly copied as-is from the GitHub UI's master template, so happy to amend.

Reply:
OK then let's leave it. It's probably just how I was reading it in my head